docs/components/App.js

```diff
@@ -41,7 +42,9 @@ export default function App({ children }: Props): Node {
       window.gtag('config', 'UA-12967896-44', {
         page_path: window.location.pathname + window.location.search + window.location.hash,
       });
+      setIsHomePage(window?.location?.pathname === '/home');
```

Comment:
@dangerismycat Is there a better way to do this?

Reply:
I'm not sure. This works for now at least